pandas/tests/reshape/test_crosstab.py

```diff
@@ -698,3 +698,46 @@ def test_margin_normalize(self):
             names=["A", "B"],
         )
         tm.assert_frame_equal(result, expected)
+
+    def test_crosstab_multiple_columns_normalize(self):
```

Comment:
maybe rename the test a bit to reflect `normalize` and `margin` set to `True`? something like `test_crosstab_multiple_columns_normalize_margin`

and add a reference for the issue number below for future reference? similar in other tests


> What do you mean by would the results be the expected transposed??

i think @simonjayhawkins meant if it's possible to integrate this test into the existing test by parametrization so that you could re-use the `df` (or other reusable vars) defined in that test. It's fine to keep as is in a separate test to me, but depends on his preference.

Reply:
I added my test to the already existing `test_margin_normalize`